techniques/css/C31.html

```diff
@@ -103,26 +99,26 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
     
     
     <section id="description">
-      <h2>Description</h2>
-      <p>The objective of this technique is to be able to present content without introducing a horizontal scroll bar at a width equivalent to 320 CSS pixels, or a vertical scroll bar at a height equivalent to 256 CSS pixels for text intended to scroll horizontally. This is done by using layout techniques that adapt to the available viewport space.</p>
-      <p><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout">Flexbox layouts</a> define layout regions that reflow as needed to display the region on the screen. Although the exact layout therefore varies, the relationship of elements and the reading order remains the same when done right. This is an effective way to create designs that present well on different devices and for users with different zoom preferences.</p>
-      <p>The basic principles of flexbox layouts are to:</p>
+      <h2>解説</h2>
+      <p>この達成方法の目的は、320 CSS ピクセルに相当する幅の水平スクロールバー、又は水平スクロールするテキストの 256 CSS ピクセルに相当する高さの垂直スクロールバーを導入せずにコンテンツを表示できるようにすることである。これには、使用可能なビューポートの空間に適応するレイアウト手法が使用される。</p>
+      <p><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout">Flexbox レイアウト</a>は、画面に領域を表示するために必要に応じてリフローするレイアウト領域を定義する。したがって、リフローが適切に行われた場合、厳密なレイアウトは異なるが、要素の関係と読み取り順序は同じままである。これは、さまざまなデバイス及びさまざまなズーム設定を持つ利用者に適したデザインを作成するための効果的な方法である。</p>
+      <p>Flexbox レイアウトの基本原則は次のとおり:</p>
       <ol>
-        <li>Define the size of layout regions using flexbox properties and media queries for specific viewport sizes, so they enlarge, shrink or wrap in the available space and respond to zoom levels;</li>       
-         <li>Position the layout regions in the flexbox container as a row of adjacent flexbox items, which may wrap to new rows as needed in much the same way as words in a paragraph wrap.</li>
+        <li>Flexbox プロパティ及びメディアクエリを使用して特定のビューポートサイズのためにレイアウト領域のサイズを定義し、利用可能なスペースで拡大、縮小、または折り返して、ズームレベルに対応する。</li>       
+         <li>Flexbox コンテナ内のレイアウト領域を、隣接する Flexbox アイテムの行として配置する。これは、段落の折り返しの単語とほぼ同じ方法で、必要に応じて新しい行に折り返すことができる。</li>
       </ol>
 
       <div class="note">
-				<p class="note-title marker">Note</p>
-				<p>Flexbox has the possibility to cause a keyboard navigation disconnect by using the order and reverse properties. The <a href="https://www.w3.org/TR/css-flexbox-1/#order-accessibility">CSS Flexible Box Layout module warns</a> against resequencing content logic as they cause incorrect source ordering and are non-conforming.</p>
+				<p class="note-title marker">注記</p>
+				<p>Flexbox では、order プロパティと reverse プロパティを使うことでキーボードナビゲーションの分断を引き起こす可能性がある。不正なソースの順序付けを引き起こし不適合なコンテンツロジックの順序変更を、<a href="https://www.w3.org/TR/css-flexbox-1/#order-accessibility">CSS Flexible Box Layout module モジュールは警告する</a>。</p>
```

Comment:
「モジュール」は (英語表記の module と重複するので) 不要かと思います。
```suggestion
				<p>Flexbox では、order プロパティと reverse プロパティを使うことでキーボードナビゲーションの分断を引き起こす可能性がある。不正なソースの順序付けを引き起こし不適合なコンテンツロジックの順序変更を、<a href="https://www.w3.org/TR/css-flexbox-1/#order-accessibility">CSS Flexible Box Layout module は警告する</a>。</p>
```